### ISSUE TITLE
Disable rotation of the plugin

### ISSUE THREAD

**Issue by alexandruantonica** [NativeScript/nativescript-imagepicker#151]
Hi,
I disabled rotation of the phone in my application, I use the app only in portrait mode.
When I use `nativescript-imagepicker` and pick an image, everything is fine until by mistake a shake the phone or change the direction to landscape. In this moment, the image picker shows the images in a new layout, instead of 4 picture is 8 pictures.

How can I disable this rotation ?

![simulator screen shot - iphone x - 2018-01-31 at 09 30 37](https://user-images.githubusercontent.com/12050715/35627008-7b153c04-068f-11e8-9c8e-1992a17a881f.png)
![simulator screen shot - iphone x - 2018-01-31 at 09 30 54](https://user-images.githubusercontent.com/12050715/35627010-7b2bfea8-068f-11e8-970e-c488fd646e50.png)

**Comment by NickIliev** [comment]
@alexanderfilipov on what iOS device/simulator you are experiencing this (iOS version) - I have tried to reproduce the issue using the demo project but to no avail. Please let me know if you have a sample project or code snippet which can reproduce the issue on specific device/ iOS version.

**Comment by tsonevn** [comment]
closing due to inactivity

**Comment by alexandruantonica** [comment]
Hi @NickIliev 
I tried with : 
- emulators : iPhone 7,8,X iOS 11.2
- real device : iPhone 7 iOS 11.2

I just checked the code and [images.ios.ts](https://github.com/NativeScript/nativescript-imagepicker/blob/master/src/images.ios.ts#L21) changes the `spanCount` of `listViewLayout` based on the layout of the phone. 

My big question is: Can we add an option to disable that? 
Sounds like a feature request but I really need this feature.

Thanks a lot :)
